lib/pharos/host/el7/el7.rb

```diff
@@ -69,6 +69,10 @@ def configure_container_runtime
         end
       end
 
+      def configure_container_runtime_safe?
+        true
```

Comment:
Why is EL based distros always safe?

Reply:
I didn't yet come up with logic how to resolve versions etc.. it's not worse than the current version.

Reply:
Fixed.